echo-pipelinetriggers/src/main/java/com/netflix/spinnaker/echo/pipelinetriggers/PipelineCache.java

```diff
@@ -132,6 +145,40 @@ void pollPipelineConfigs() {
     }
   }
 
+  private List<Pipeline> fetchHydratedPipelines() {
+    List<Map> rawPipelines = front50.getPipelines();
+    if (rawPipelines == null) {
+      return Collections.EMPTY_LIST;
```

Comment:
nit: `Collections.emptyList()` uses type inference to return `List<Pipeline>` whereas `Collections.EMPTY_LIST` is a raw list.  Not super important for an empty list...

Reply:
Done.